SIL.WritingSystems/Resources/LanguageIndex.txt

```diff
@@ -15699,37 +15699,37 @@ gaw	PG	LA	Butelkud-Guntabak
 gaw	PG	LA	Garuh
 gaw	PG	LA	Nobanob
 gaw	PG	LA	Nobnob
-gax	ET	L	Oromo, Borana-Arsi-Guji
-gax	ET	LA	Afan Oromo
-gax	ET	LA	Southern Oromo
-gax	ET	LP	Galligna
-gax	ET	LP	Gallinya
-gax	KE	D	Arsi
-gax	KE	D	Boran
-gax	KE	D	Guji
-gax	KE	L	Borana
-gax	KE	LA	Booran
-gax	KE	LA	Boraan
-gax	KE	LA	Boran
-gax	KE	LA	Oromo
-gax	KE	LA	Southern Oromo
-gax	SO	D	Borana
-gax	SO	DA	Booran
-gax	SO	DA	Boran
-gax	SO	L	Oromo, Borana-Arsi-Guji
-gax	SO	LA	Southern Oromo
+gax	ET	!L	Oromo, Borana-Arsi-Guji
```

Comment:
Shouldn't you go ahead and add back in the lines you deleted yesterday and put the bang on them?